tests/testthat/test-get_followers.R

```diff
@@ -0,0 +1,5 @@
+test_that("get_followers works", {
+  kfc <- get_followers("KFC")
+  expect_s3_class(kfc, "tbl")
+  expect_equal(nrow(kfc), 5000)
```

Comment:
I'd suggest setting `n = 100` or similar, just to keep the tests as fast as possible

Reply:
Yeah, did so with others but missed this one.